index.js

```diff
@@ -128,13 +129,21 @@ export const Link = (props) => {
   return cloneElement(jsx, extraProps);
 };
 
+const flattenChildren = (children) => {
+  return Array.isArray(children)
+    ? children.flatMap((c) =>
```

Comment:
I wonder if it is safe to use `flatMap` here. I know it's widely supported, but it's part of ES2019. wouter currently targets ES2015, would that maybe better to use a short polyfill: `var flatMap = (arr, cb) => [].concat(...arr.map(cb))`

Reply:
But overall, this is quite smart! 👍

Reply:
Ah yes, did not think of that, so used to Typescript/ESNext, I always struggle in build-less environments.
I'll add the polyfill you suggested tomorrow. It will increase the size by a few bytes but not that much I guess.

Reply:
I've added the polyfill. I inlined it in `flattenChildren` directly as this was best for bundle size. Bundle size increase was 5 bytes for the polyfill.